From 803cbdfbd78b66b17ead45b1584d65a258e785bf Mon Sep 17 00:00:00 2001
From: Ray Strode <rstrode@redhat.com>
Date: Tue, 24 May 2011 20:58:10 -0400
Subject: [PATCH] ck-history: don't truncate --frequent output to 8 chars

GDM screenscrapes the output and so we need the full username.

https://bugzilla.gnome.org/show_bug.cgi?id=650330
---
 tools/ck-history.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/ck-history.c b/tools/ck-history.c
index d02caaa..85d9e6f 100644
--- a/tools/ck-history.c
+++ b/tools/ck-history.c
@@ -804,7 +804,7 @@ generate_report_frequent (int         uid,
                 data = user_counts->data;
 
                 username = get_user_name_for_uid (data->uid);
-                g_print ("%-8.8s %u\n", username, data->count);
+                g_print ("%-8s %u\n", username, data->count);
                 g_free (data);
                 user_counts = g_list_delete_link (user_counts, user_counts);
                 g_free (username);
-- 
1.7.9.4

